test/parallel/test-fs-fchmod.js

```diff
@@ -46,8 +46,8 @@ const fs = require('fs');
   const errObj = {
     code: 'ERR_OUT_OF_RANGE',
     name: 'RangeError [ERR_OUT_OF_RANGE]',
-    message: 'The value of "mode" is out of range. It must be >= 0 && < ' +
-             `4294967296. Received ${input}`
+    message: 'The value of "mode" is out of range. It must be >= 0 && <= ' +
```

Comment:
is this necessary?

Reply:
oh, this is a test, so the error is now coming from validateInt32() and it's different, ignore I guess